packages/client/helpers/build.ts

```diff
@@ -28,18 +26,16 @@ const browserBuildConfig: BuildOptions = {
 }
 
 // we define the config for proxy
-const proxyBuildConfig: BuildOptions = {
-  name: 'proxy',
+const edgeRuntimeBuildConfig: BuildOptions = {
+  name: 'edge',
   entryPoints: ['src/runtime/index.ts'],
-  outfile: 'runtime/proxy',
+  outfile: 'runtime/edge',
```

Comment:
Renaming the runtime to `edge` for `@prisma/client/edge`. Data Proxy can still be used with `@prisma/client` on Node.